lucet-runtime/lucet-runtime-tests/src/host.rs

```diff
@@ -119,7 +119,7 @@ macro_rules! host_tests {
             let mut inst = region
                 .new_instance(module)
                 .expect("instance can be created");
-            inst.run(b"main", &[Val::GuestPtr(0), Val::I32(0)])
+            inst.run(b"main", &[0u32.into(), 0i32.into()])
```

Comment:
this actually changes the first argument from `Val::GuestPtr(0)` to `Val::U32(0)`; was this intentional?